Software/vortex_ws/src/sensors/package.xml

```diff
@@ -5,12 +5,13 @@
   <version>0.0.0</version>
   <description>sensors interfaces and fusion package</description>
   <maintainer email="info@vortex-co.com">vortex-co</maintainer>
-  <license>GNU GENERAL PUBLIC LICENSE</license>
+  <license>Apache License 2.0</license>
```

Comment:
This replacement is performed to match the license header in the source code, Currently I'm searching for a license header for GGP license or testing if we could put GGP license in the package.xml file and keep the Apache 2 license header in the source code.